woocommerce/migrator-cli
# Refactor orders and product import commands

**Description**
This PR moves the code to import orders and product into their own files so it will be easier for future development

**Testing steps**
- If you have already ran an import before, remove the orders, customers and products first
- Execute the products import with `wp migrator products`
- Make sure it runs successfully
- Execute the order imports `wp migrator orders`
- Make sure it runs successfully
- Execute the order tags import `wp migrator fix_missing_order_tags`
- Make sure it runs successfully
